pkg/producer/http.go

```diff
@@ -0,0 +1,32 @@
+package producer
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/asecurityteam/nexpose-vuln-notifier/pkg/domain"
+)
+
+// AssetProducer holds streaming appliance configuration
+type AssetProducer struct {
+	HTTPClient *http.Client
+	Endpoint   string
+}
+
+// Produce publishes sends the asset event to the streaming appliance
+func (p *AssetProducer) Produce(ctx context.Context, asset domain.AssetEvent) error {
+	body, _ := json.Marshal(asset)
```

Comment:
None of the fields in the `domain.AssetEvent` has JSON annotations, so the outgoing struct will be capital case, which doesn't match with the snake case define in our docs.

Reply:
Thanks for pointing this out. I'll make the change